control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -751,6 +754,12 @@ func (r *reconciler) Reconcile(ctx context.Context, _ ctrl.Request) (ctrl.Result
 		}
 	}
 
+	// Reconcile GlobalPullSecret
+	hccoImage := os.Getenv("HOSTED_CLUSTER_CONFIG_OPERATOR_IMAGE")
```

Comment:
why is the image a param to reconcileGlobalPullSecret? can't we just set it in the deployment definition?